frontend/scripts/react-components/shared/years-range-dropdown/years-range-dropdown.component.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Dropdown from 'react-components/shared/dropdown';
+import YearsRange from './years-range/years-range.component';
+
+function YearsRangeDropdown(props) {
+  const { years, selectedYears, selectYears, ...rest } = props;
+  const multipleYears = selectedYears[0] !== selectedYears[1];
```

Comment:
It would be nice to change the interaction so we don't have to click one year twice to select a single year. Just a reminder